### PR TITLE
Updating Readme

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

PR for issue https://github.com/eclipse/codewind/issues/368.

### CODE REVIEW

**Comment by sishida** [comment]
@eharris369 Hi, Erin, Could you please review the accuracy of these updates?

I based the changes off of the VS Code README, but I understand that VS Code and Eclipse can be pretty different.

Also, does Eclipse have a Workspace Welcome Page that we could link to? Thank you!

File: README.md [hunk 1]
```diff
@@ -1,34 +1,41 @@
...
 Prerequisites
 - Download and install the latest [Eclipse IDE for Java EE Developers](https://www.eclipse.org/downloads/packages/release/) or use an existing installation. The earliest supported version of the Eclipse IDE for Codewind for Eclipse is 4.11.0 (2019-03).
 - Install Docker.
 - If you use Linux, you also need to install Docker Compose.
 
 Complete the installation:
 1. Install [Codewind from the Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind).
-2. Open the **Codewind Explorer** view.
+2. Open the **Codewind Explorer** view. Go to **Window**>**Show View**>**Other…**>**Codewind**>**Codewind Explorer**.
 3. Double-click the **Codewind** entry in the view to finish installing Codewind. The download is approximately 1 GB. For more information, see [Installing Codewind for Eclipse](https://www.eclipse.org/codewind/mdteclipseinstallinfo.html).
```
Comment (eharris369) [inline README.md]: Looks like the link needs updating to: https://www.eclipse.org/codewind/mdt-eclipse-installinfo.html.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,41 @@
...
-- Set up a Chrome debug session for **Node.js** projects.
-- View application and build logs in the **Console** view.
-- View and edit project deployment information.
-- Open a shell session into a Codewind application container.
-- Toggle project auto build and manually initiate project builds.
-- Integrate Codewind validation errors into the **Markers** view.
-- Disable, enable, and remove projects.
+[Features:](https://www.eclipse.org/codewind/mdteclipsemanagingprojects.html)</br>
+- **Open Application**: Opens the application in the default Eclipse browser. This action is only available when the application is running or debugging.
+- **Open Project Overview**: Opens the overview page for a project. You can use this action to see information about the project, enable or disable the project, turn auto build on and off, or edit project settings.
```
Comment (eharris369) [inline README.md]: You can no longer enable/disable the project or turn auto build on/off from the overview page in Eclipse.  You can still edit the project settings.

File: README.md [hunk 3]
```diff
@@ -1,34 +1,41 @@
...
+- **Open Container Shell**: Opens a shell into your application container. This action is available only when the container is active.
+- **Open Application Monitor**: Opens the application monitor in the default Eclipse browser. Use this action to monitor the activity and health of your application. This action is available only when the application is running or debugging.
+- **Import Project**: Imports your project into the Eclipse workspace.
+- **Show Log Files**: If log files are available, this action displays a list of log files. In the Eclipse **Console** view, click a log file to open it, or click **Show All** to open all available log files. The individual log files are toggle actions. Click the log file again to remove the log file, or click **Hide All** to remove all log files from the **Console** view.
```
Comment (eharris369) [inline README.md]: If the user clicks Show All or one of the individual log file toggle actions then the log file will open in the Eclipse Console view.  So this is not correct: "In the Eclipse **Console** view, click a log file to open it, or click **Show All** to open all available log files."

File: README.md [hunk 4]
```diff
@@ -1,34 +1,41 @@
...
+- **Build**: Initiate a build of your project. This action is not available if a build is already running. For more information, see [Building Codewind projects](https://www.eclipse.org/codewind/mdteclipsebuildproject.html).
+- **Disable Auto Build**: Use this to disable automatic builds if you are making a lot of changes and don't want builds to be triggered until you are done. This action is available only when auto build is enabled.
+- **Enable Auto Build**: Use this to re-enable automatic builds whenever a change is made. This action is available only when auto build is disabled.
+- **Remove**: Removes a project. This action removes the project from Codewind. You can then use Eclipse to delete the project from the Eclipse workspace and the file system.
+- **Refresh**: If the project gets out of sync, use this option to refresh it. To refresh all projects, right-click on the **Local Projects** item in the **Codewind Explorer** view and select **Refresh**.
```
Comment (eharris369) [inline README.md]: It is now called "Projects (Local)" rather than "Local Projects"

File: README.md [hunk 5]
```diff
@@ -1,34 +1,41 @@
...
-- View and edit project deployment information.
-- Open a shell session into a Codewind application container.
-- Toggle project auto build and manually initiate project builds.
-- Integrate Codewind validation errors into the **Markers** view.
-- Disable, enable, and remove projects.
+[Features:](https://www.eclipse.org/codewind/mdteclipsemanagingprojects.html)</br>
+- **Open Application**: Opens the application in the default Eclipse browser. This action is only available when the application is running or debugging.
+- **Open Project Overview**: Opens the overview page for a project. You can use this action to see information about the project, enable or disable the project, turn auto build on and off, or edit project settings.
+- **Open Container Shell**: Opens a shell into your application container. This action is available only when the container is active.
+- **Open Application Monitor**: Opens the application monitor in the default Eclipse browser. Use this action to monitor the activity and health of your application. This action is available only when the application is running or debugging.
```
Comment (eharris369) [inline README.md]: There is also the Open Performance Dashboard action.

File: README.md [hunk 6]
```diff
@@ -1,34 +1,41 @@
...
+- **Restart in Debug Mode**: Restarts the application in debug mode and attaches the debugger. Only MicroProfile/Java EE, Spring, and Node.js projects can be debugged. For more information, see [Debugging Codewind projects](https://www.eclipse.org/codewind/mdteclipsedebugproject.html).
+- **Attach Debugger**: If you detached the debugger accidentally or restarted Eclipse, use this action to re-attach the debugger to an application in debug mode. For more information, see [Debugging Codewind projects](https://www.eclipse.org/codewind/mdteclipsedebugproject.html).
+- **Build**: Initiate a build of your project. This action is not available if a build is already running. For more information, see [Building Codewind projects](https://www.eclipse.org/codewind/mdteclipsebuildproject.html).
+- **Disable Auto Build**: Use this to disable automatic builds if you are making a lot of changes and don't want builds to be triggered until you are done. This action is available only when auto build is enabled.
+- **Enable Auto Build**: Use this to re-enable automatic builds whenever a change is made. This action is available only when auto build is disabled.
```
Comment (eharris369) [inline README.md]: There is also the Disable Project action

**Pull request merged**